README.md

```diff
@@ -1 +1,210 @@
+  <ul>
+    <li><a href="https://css-tricks.com/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The key features added in this project.
+
+- Mobile Version
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get started as a deveoper, below is instructions on how to make use of this project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Latest version of your browser
+2. Code Editor
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command
+
+```sh
+  cd Folder_Name
+  git clone git@github.com:manzitresor/Portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Portfolio
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using liver server extension.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> This is a solo project, successfully submitted by myself;
+
+👤 Manzi Tresor
+
+- GitHub: [@githubhandle](https://github.com/manzitresor)
+-Twitter: [@githubhandle](https://twitter.com/MANZITresor3)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/manzi-tresor-783b4022a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added in this project include.
+
+- Home Page For mobile version
+- About Page for mobile vesrion
+- Desktop Version
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to Reach Out To me 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please support me by following me on Github; <a href ="https://github.com/manzitresor/Hello_Microverse.git">Manzi</a> and give a star to this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team for making this program possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Frequently Asked Questions
+
+- Which browser works best for this project?
+
+  - Any modern browser i.e.  Google Chrome,Firefox,Microsoft Edge, Apple Safari, Opera
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please fix this broken link in your Readme, add a license file and make sure the MIT license link is not broken :+1: